fiftyone/utils/coco.py

```diff
@@ -1,15 +1,20 @@
 """
 Utilities for working with datasets in
-`COCO format <http://cocodataset.org/#home>`_.
+`COCO format <https://cocodataset.org/#format-data>`_.
 
 | Copyright 2017-2020, Voxel51, Inc.
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
 from collections import defaultdict
 from datetime import datetime
+from itertools import groupby
 import logging
 import os
+import warnings
+
+import numpy as np
+from skimage import measure
```

Comment:
`skimage` appears to be a new dependency - is it optional? It's causing zoo documentation generation to fail now, so it should at least be documented somewhere, even if it's not required.
https://github.com/voxel51/fiftyone/runs/1250898309#step:9:176

Reply:
Yes, I forgot to mention that in my review. My mistake.